travel_platform/izi_travel_client.py

```diff
@@ -119,7 +120,7 @@ def find_city_by_name(self, query):
         city = {}
 
         if response:
-            city_data = response[0]
+            city_data = response.pop(0)
```

Comment:
Well, you don't need index here (if there is no more than 1 item in list).
If there is more than one item in list, why do we need just one of them?